.github/workflows/release.yml

```
@@ -1,106 +1,16 @@
-name: release
+name: Release Buildpacks
```

(this repo only has one buildpack, and that's unlikely to change)
```suggestion
name: Release Buildpack
```